students/elmar_m/lesson03/list_lab.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+
+'''
+file: list_lab.py
+elmar_m / 22e88@mailbox.org
+Lesson03: List Lab Exercise
+'''
+
+def print_list(l):
+    '''
+    prints the elements of a list in one line, ending with a 
+    single line feed
+    '''
+    for i in l:
+        print(i, '', end='')     
+    print()
```

Comment:
I like the utility print function 👍 

Here is alternate as a one liner: `print(' '.join(map(str, l)))` (note that `map` isn't necessary in your case since your fruits are all strings.